doc/assist.md

```diff
@@ -0,0 +1,155 @@
+* if s1(x1) address of storage area and s2 length of area are specified just dump that area.
+
+### XGET - read record from ASCII file
+
+Format: `RXSS X'E0Axbdddbddd' s1(x1),s2`
+
+* Read record from ASCII file DDNAME=XGET into area s1(x1) with length of s2.
+* Requires that the environment variable `XGET` be set and point to desired input file.
+* If a file operation is successful the condition code is set to 0.
+* At end of file the condition code 1 is set.
+* If a file open error occurs, program terminates with S013 abend with error message showing the file specification which failed.
+* If any other error occurs such as missing length, condition code 2 is set.
+
+### XHEXI - convert hex to binary
+
+Format: `RX X'61' r1,s2` 
+
+Convert hex to binary (cc3 if no hex, update field addr).
+
+### XHEXO - convert binary to hex
+
+Format: `RX X'62' r1,s2` 
+
+convert binary to hex (12 bytes, update field addr).
+
+### XLIMD - Set default dump storage area
+
+Format: `RXSS X'E08xrbdddbddd' s1(x1),s2`
+
+Set default XDUMP storage area address and length.
+
+### XPNCH - Write to punch
+
+Format: `RXSS X'E04xbdddbddd' s1(x1),s2`
+
+* Write to DDNAME=XPNCH with length s2.
+* Requires that the environment variable `XPNCH` be set and point to desired output file.
+* Does not set condition code.
+* If a file error occurs the program aborts with abend code S013.
+* If a file open error occurs, program terminates with S013 abend with error message showing the file specification which failed.
+
+### XPRNT - Write to print
+
+Format: `RXSS X'E02xbdddbddd' s1(x1),s2`
+
+* Write to DDNAME=XPRNT with length s2.
+* Requires that the environment variable `XPRNT` be set and point to desired output file.
+* Does not set condition code.
+* If a file error occurs, the program aborts with abend code S013.
+* If a file open error occurs, program terminates with S013 abend with error message showing the file specification which failed.
+
+### XPUT - Write to XPUT
+
+Format: `RXSS X'E0Cxbdddbddd' s1(x1),s2`
+
+* Write to DDNAME=XPUT for length s2.
+* Requires that the environment variable `XPUT` be set and point to desired output file.
+* If a file operation is successful the condition code is set to 0.
+* If a file open error occurs, program terminates with S013 abend with error message showing the file specification which failed.
+* If any other error occurs such as missing length, condition code 2 is set.
+
+### XREAD - Read from XREAD
+
+Format: `RXSS X'E00xbdddbddd' s1(x1),s2` 
+
+* Read record from DDNAME=XREAD for length s2.
+* Requires that the environment variable `XREAD` be set and point to desired input file.
+* If a file operation is successful the condition code is set to 0.
+* At end of file, the condition code 1 is set.
+* If a file error occurs, the program aborts with abend code S013.
+* If a file open error occurs, program terminates with S013 abend with error message showing the file specification which failed.
```

Comment:
OK.. so there is already a link to the NIU manual at the top of the ASSIST page. I have made this more obvious. The question then is, when do you use a footnote vs when do you use a link inline. The footnote was used as it was providing the source of details regarding the origin of ASSIST, whereas the link is just to the manual and I thought deserved a link inline rather than a footnote. We can provide guidance in the doc contribution guide if required.. just need to justify why this seems right.. or why you think this is wrong.. Happy for discussion.

Reply:
@dsh33782 and @jyganci are comfortable with the current link and footnotes.. resolving.